Documentation/ApiOverview/ContentElements/MigrationListType.rst

```diff
@@ -0,0 +1,120 @@
+..  include:: /Includes.rst.txt
+..  _plugins-list-type:
+..  _plugins-list-type-migration:
+
+=========================================
+Migration: `list_type` plugins to `CType`
+=========================================
+
+..  deprecated:: 13.4
+    The plugin content element (:php:`list`) and the plugin sub types
+    field (:php:`list_type`) have been marked as deprecated in TYPO3 v13.4 and
+    will be removed in TYPO3 v14.0.
+
+Several steps are important in the migration from `list_type` plugins to `CType`
+plugins:
+
+*   Register plugins using the `CType` record type
+*   Create update wizard which extends :php:`\TYPO3\CMS\Install\Updates\AbstractListTypeToCTypeUpdate`
+    and add :php:`list_type` to :php:`CType` mapping for each plugin to migrate.
+*   Migrate possible FlexForm registration and add dedicated :php:`showitem` TCA
+    configuration
+*   Migrate possible PreviewRenderer registration in TCA
+*   Adapt possible content element wizard items in Page TSConfig, where
+    :php:`list_type` is used
+*   Adapt possible content element restrictions in backend layouts or container
+    elements defined by third-party extensions like ext:content_defender
+
+..  contents:: Table of content
+
+..  _plugins-list-type-migration-extbase:
+
+Migration example: Extbase plugin
+=================================
+
+..  _plugins-list-type-migration-extbase-configuration:
+
+1. Adjust the Extbase plugin configuration
```

Comment:
What about using: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Howto/WritingReST/CheatSheet.html#ref-styled-numbered-lists-big-nums

Reply:
I woul like to have the single steps linkable and that does not work with a bignum list